Release PR for public release r1.2 (M4) (#98)
camaraproject/ConnectivityInsights: CHANGELOG.md

* for a public API version, the consolidated changes since the release of the
previous public API version

# r1.2
## Release Notes

This release contains the definition and documentation of
* connectivity-insights v0.4.0
* connectivity-insights-subscriptions v0.4.0
* application-profiles v0.3.0

The API definition(s) are based on
* Commonalities v0.4.0
* Identity and Consent Management v0.2.0

> *  **r1.2 release of connectivity insights has the following API definitions:**

Reviewer: ```suggestion
*  **r1.2 release of ConnectivityInsights has the following API definitions:**
```

    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml&nocors)
    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml)
    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml)

    *  **application-profiles v0.3.0**
    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml&nocors)
    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml)
    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml)

### Added
* n/a
### Changed
* n/a

> **Full Changelog**: https://github.com/camaraproject/connectivityInsights/compare/r1.1...r1.2

Reviewer: This line - if included - should reflect all changes since the last public release, not compared to last pre-release. In your case - compared to no previous release - it would be:

```suggestion
**Full Changelog**: https://github.com/camaraproject/connectivityInsights/commits/r1.2
```

* Application Profiles API v0.3.0

The API definition(s) are based on
* Commonalities v0.4.0
* Identity and Consent Management v0.2.0

## Connectivity Insights API v0.4.0

> 0.4.0-rc.1 is the first release of the API. The version is aligned with

Reviewer 2: ```suggestion
0.4.0 is the first release of the API. The version is aligned with
```

* Application Profiles API v0.3.0

The API definition(s) are based on
* Commonalities v0.4.0
* Identity and Consent Management v0.2.0

*  **r1.2 release of connectivity insights has the following API definitions:**

>     *  **Connectivity Insights API v0.4.0**

Reviewer 3: I suggest to adhere to the https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/CHANGELOG_TEMPLATE.md
It uses `api-name` so _connectivity-insights_ instead of _Connectivity Insights API_ and so on.
There should be separate sections for each API as in r1.1

Author: issue #100 created to track this

    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml)
    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml)

    *  **Application Profiles API 0.3.0**
    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml&nocors)
    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml)
    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml)

### Added
* n/a
### Changed
* n/a

**Full Changelog**: https://github.com/camaraproject/connectivityInsights/compare/r1.1...r1.2

# r1.1
## Release Notes

This release contains the definition and documentation of

> * Connectivity Insights API v0.4.0
> * Connectivity Insights Subscriptions API v0.4.0
> * Application Profiles API v0.3.0

Reviewer 3: The section related to r1.1 should stay unchanged and in-line with https://github.com/camaraproject/ConnectivityInsights/releases/tag/r1.1

```suggestion
* Connectivity Insights API v0.4.0 - rc.1 
* Connectivity Insights Subscriptions API v0.4.0 - rc.1
* Application Profiles API v0.3.0 - rc.1
```

Author: issue #99 created to track this.

### Added
* n/a
### Changed
* n/a
## Connectivity Insights Subscriptions API v0.4.0

> 0.4.0-rc.1 is the first release of the API. The version is aligned with

Reviewer 2: ```suggestion
0.4.0 is the first release of the API. The version is aligned with
```

### Added
* n/a
### Changed
* n/a

## Application Profiles API 0.3.0

> 0.3.0-rc.1 is the first release of the API. The version is aligned with

Reviewer 2: ```suggestion
0.3.0 is the first release of the API. The version is aligned with
```
